tests/graph/test_compute_test_value.py

```diff
@@ -85,7 +85,15 @@ def test_variable_only(self):
         z = dot(x, y)
         assert hasattr(z.tag, "test_value")
         f = pytensor.function([x, y], z)
-        assert _allclose(f(x.tag.test_value, y.tag.test_value), z.tag.test_value)
+        atol_, rtol_ = _get_atol_rtol(
```

Comment:
If this ends up only being used in the tests here, we can remove from math and include only in the test file?